index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles.css" />
+    <title>Home</title>
+  </head>
+  <body>
+   <div class="wrapper">
+     <header class="toolbar">
+       <div class="logo">
+         <img src="./image/images-head/logo.png" alt="MyLogo" />
```

Comment:
- [ ] This logo it's not only an image but a link, this is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info

Reply:
Thank you for the recommendation. link added for it to be directed to the home page.